### ISSUE TITLE
Color contrast not 7.1

### ISSUE THREAD

**Issue by philipashlock** [GSA/datagov-wptheme#206]
Text links under the icons have insufficient contrast. Expected a contrast ratio of at least 7.1.

**Comment by Kapil-bulchandani** [comment]
attached are the shades of color we use we can go darker to get the contrast ratio over 7.1
![screen shot 2014-04-29 at 11 01 45 am](https://cloud.githubusercontent.com/assets/2903623/2830449/c23ca8fe-cfae-11e3-932b-efb3b51a6e11.png)

**Comment by Kapil-bulchandani** [comment]
We have used #275379 for the topics under home page, this color passes the contrast ratio

**Comment by dialsunny** [comment]
The new color indicated above has been added.
